data/7/score.mei

```diff
@@ -69,7 +70,7 @@
                             <staffDef xml:id="mattheson" clef.shape="G" clef.line="2" key.mode="major" key.sig="2s" meter.count="12" meter.unit="8" n="1" lines="5">
                                 <label>Mattheson's annotations</label>
                             </staffDef>
-                            <staffDef xml:id="pfeffer" clef.shape="G" clef.line="2" key.mode="major" key.sig="2s" meter.count="12" meter.unit="8" n="41" lines="5">
+                            <staffDef resp="#pfeffer" xml:id="pfeffered" clef.shape="G" clef.line="2" key.mode="major" key.sig="2s" meter.count="12" meter.unit="8" n="41" lines="5">
```

Comment:
Sorry, I just saw this now: At the moment, hiding and showing of staves is based on the `xml:id`s of the staffDefs, so this will actually break a correct score displaying.
In future I'd like to have realizations etc. anyways separated from the actual bass file itself, so it's probably going to be removed anyways. But for the moment I wouldn't be happy about breaking functionality …

Reply:
ok, I've reverted the commit.